src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import tokenService from 'services/tokenService';
+import classes from './ProfileInfo.module.scss';
+
+interface UserData {
+  email: string;
+  firstName: string;
+  lastName: string;
+  address: string;
+  phoneNumber: string;
+}
+
+const ProfileInfo = () => {
+  const [user, setUser] = useState<UserData>();
+  const { getUserId } = tokenService;
+  const id = getUserId();
```

Comment:
It could be replaced with `const id = tokenService.getUserId();`

Reply:
No this should stay as it is